extensions/amp-list/0.1/test-e2e/test-load-more-auto.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+describes.endtoend('AMP list load-more=auto', {
+}, async env => {
+  const pageWidth = 800;
+  const pageHeight = 420;
+  let controller;
+  let ampDriver;
+
+  beforeEach(async() => {
+    controller = env.controller;
+    ampDriver = env.ampDriver;
+
+    await controller.navigateTo('http://localhost:8000/test/manual/amp-list/load-more-auto.amp.html');
+    await ampDriver.toggleExperiment('amp-list-load-more', true);
+
+    await controller.setWindowRect({
+      width: pageWidth,
+      height: pageHeight,
+    });
+    await controller.navigateTo(
+        'http://localhost:8000/test/manual/amp-list/load-more-auto.amp.html');
+  });
+
+  it('should render correctly', async() => {
+    const listItems = await controller.findElements('.item');
+    expect(listItems.length).to.equal(2);
+
+    const loader = await controller.findElement('[load-more-loading]');
+    expect(loader).to.not.be.null;
```

Comment:
I think you can also do `expect(loader).to.be.ok`, I seem to get a bit tripped up reading `to.not.be.null` personally.